src/App.js

```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```

Comment:
Replace `'react'` with `"react"` (prettier/prettier)

```suggestion
import React from "react";
```